modules/openapi/component-protocol/scenarios/home-page-content/components/emptyOrgContent/render.go

```diff
@@ -114,7 +114,7 @@ func (e *EmptyOrgContent) Render(ctx context.Context, c *apistructs.Component, s
 	}, Value{
 		Props: PropValue{
 			RenderType: "linkText",
-			Title:      i18nLocale.Get(i18n.I18nKeyOrgCreate),
+			Title:      fmt.Sprintf("* %s", i18nLocale.Get(i18n.I18nKeyOrgCreate)),
```

Comment:
paste an screenshot in comment, let me know how it looks like

Reply:
![image](https://user-images.githubusercontent.com/30427474/124856444-194a5380-dfdd-11eb-875d-bfffbac0c04e.png)